src/HonzaBotner.Discord/Command/Permissions/Data/IPermissionEntry.cs

```diff
@@ -0,0 +1,13 @@
+using System;
+
+namespace HonzaBotner.Discord.Command.Permissions.Data
+{
+    public interface IPermissionEntry
```

Comment:
This doesnt have to be interface. Sealed class would suffice, since it will be just data transportation object (DTO).